### ISSUE TITLE
Broken tests in satpiler

### ISSUE THREAD

**Issue by cernoch** [master-keying/mks#4]
The introduction of CMake brought a problem with `satpiler`.

**Platform:** Ubuntu 16.04

**Steps to reproduce:**
```bash
git checkout eff5c64
rm -rf build && mkdir build && cd build
cmake .. -DMKS_USE_CPPREST=1
make -j && ./tests
```
Output (bad):
```
-------------------------------------------------------------------------------
satpiler::performance
-------------------------------------------------------------------------------
/home/radek/.cc/mks/src/test/cpp/satpiler_test.cpp:98
...............................................................................

/home/radek/.cc/mks/src/test/cpp/satpiler_test.cpp:111: FAILED:
  REQUIRE( sattor.solve() )
due to unexpected exception with message:
  Exception translation was disabled by CATCH_CONFIG_FAST_COMPILE

/home/radek/.cc/mks/src/test/cpp/satpiler_test.cpp:111: FAILED:
  {Unknown expression after the reported line}
due to unexpected exception with message:
  Unknown exception
```

**Last good commit:**
```bash
git checkout 74890ad
make -j tests WITH_REST=1
```
Output (good):
```
Test file "satpiler_test"
===============================================================================
All tests passed (60 assertions in 9 test cases)
```

**Comment by horenmar** [comment]
Works on my computer™

I would suggest removing `CATCH_CONFIG_FAST_COMPILE` from the build and checking what the exception is.

**Comment by cernoch** [comment]
New observation: Without `MKS_USE_CPPREST=1`, the tests work well.

**Comment by cernoch** [comment]
New observation: Running the problematic test only `./tests 'satpiler::performance'` works well.

...a memory leak?

**Comment by cernoch** [comment]
Or a threading issue? I'm totally lost here.

**Comment by cernoch** [comment]
Fixed in d2e5fe4abcf515e919021c4704d85f3bef2ad606.
